CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
-# v0.13.3
-
+# v0.14.0
+- New flexible interface API (#378)
```

Comment:
I think it would be good to give a short summary of what this entails for users. I.e., warn users of the flexible interface that this update is going to be a breaking change and they will need to adapt their code. We should also directly link to the tutorial that shows the updated use and potentially even include a minimal code example that shows the old interface and how one would use the new interface instead. We want breaking changes to be as non-disruptive as possible and easy to upgrade -- else it might be an annoying experience and take up time